lib/nerves_hub_device/socket.ex

```diff
@@ -65,4 +67,41 @@ defmodule NervesHubDevice.Socket do
       {:ok, decoded} -> X509.PrivateKey.to_der(decoded)
     end
   end
+
+  defp opts_from_nerves_key_or_config(user_config) do
+    cacerts = user_config[:cacerts] || Certificate.ca_certs()
+
+    with true <- Application.get_env(:nerves_hub_device, :nerves_key, false),
+         :nerves_key <- Application.get_application(NervesKey) do
+      {:ok, i2c} = ATECC508A.Transport.I2C.init([])
```

Comment:
This line and line 88 require that the NervesKey be connected to I2C bus 1. Due to a lot of luck, this is the case for all of the Raspberry Pis and our board. I'm cool with I2C-1 being the default, but is should probably be passed in so that it can be changed.

Reply:
You can now configure `:i2c_bus` and `certificate_pair` here. Also updated the README to reflect